types/ckeditor__ckeditor5-utils/src/difftochanges.d.ts

```diff
@@ -22,14 +22,16 @@ import { Change } from "@ckeditor/ckeditor5-utils/src/diff";
  */
 export default function diffToChanges(diff: Change[], output: string | string[]): Array<DeleteChange | InsertChange>;
 
-export interface DeleteChange {
+interface DeleteChange {
     index: number;
     type: "delete";
     howMany: number;
 }
 
-export interface InsertChange {
+interface InsertChange {
```

Comment:
Again, there's no reason to hide these as it makes it harder for consumers to code against them without having to duplicate the definition.